src/index.js

```diff
@@ -5,6 +5,9 @@ import devtools from 'hyperapp-redux-devtools'
 import GraphView from './graph-view-component'
 import EditorView from './editor-component'
 import {getLogger} from './logger'
+import {retrieveGraphData} from './temp-data-component'
```

Comment:
can you finalize a name. it's also not a component. a component is something that is rendered to the screen

Reply:
sorry, was a working file, will change